context/v1.jsonld

```diff
@@ -80,6 +86,7 @@
           "@type": "http://www.w3.org/2001/XMLSchema#base64Binary"
         },
         "un_distinguishing_sign": "https://w3id.org/vdl#unDistinguishingSign",
+        "url": "https://schema.org/url",
```

Comment:
Right, I might default it to just `"@type": "@id"` to reduce schema.org weirdness (but still be compatible w/ schema.org). I think the typing information is a hold over from Microdata, so it's not necessary when using w/ RDF / JSON-LD. Unless you feel strongly about this, I'll make the change to ensure this is a URL (in the JSON-LD sense of the term).

Reply:
Unfortunately I think putting the weird schema.org type is necessary here if we want LD applications to interpret the documents correctly. `"@type": "@id"` is not the same as `"@type": "https://schema.org/URL"` and it is not correct w.r.t. schema.org. The resulting RDF dataset is not the same.

```json
{
  "@context": { "@type": "https://schema.org/URL" },
  "url": "http://example.org/foo"
}
```
is equivalent to:
```
_:subject <https://schema.org/url> "http://example.org/foo"^^<https://schema.org/URL> .
```

while
```json
{
  "@context": { "@type": "@id" },
  "url": "http://example.org/foo"
}
```
is equivalent to:
```
_:subject <https://schema.org/url> <http://example.org/foo> .
```
Here `<http://example.org/foo>` is not a schema.org URL, it is a resource identifier for something that may or may not be a schema.org URL. And if its an URL it may not be `http://example.org/foo` for instance if I write:
```json
{
  "@context": { "@type": "@id" },
  "url": "http://example.org/foo",
  "http://example.org/foo": { "@value": "http://example.org/completely/different/url", "@type": "https://schema.org/URL" }
}
```
it is equivalent to:
```
_:subject <https://schema.org/url> "http://example.org/completely/different/url"^^<https://schema.org/URL> .
```

Reply:
We might have the same issue with `https://schema.org/image`. Is the value of the `image` term supposed to be a reference to a `https://schema.org/ImageObject` (in which case `"@type": "@id"` is fine), or is it supposed to be an URL to the image (in which case `"@type": "https://schema.org/URL"` should be used).